va/validation-authority.go

```diff
@@ -289,7 +288,7 @@ func (va *ValidationAuthorityImpl) fetchHTTP(identifier core.AcmeIdentifier, pat
 		}
 		return emptyBody, challenge, err
 	}
-
+	httpResponse.Body.Close()
```

Comment:
Whoops, we raced on this. I merged after I saw your tag.

You're totally right. I'll put up another branch.